docs/additional/digitalsign.adoc

```diff
@@ -56,3 +56,40 @@ dookug:
 <3> `keystore` is mandatory and specifies the location of the signing keys.
 <4> `keystorePass` and `keystoreType` specify the password and type of the keystore file, respectively. These are also mandatory.
 <5> the identifier of the private key within the keystore
+
+== Visible signature (using the EU DSS ESIGN library)
+
+In case you want to add visible (clickable) signature to the PDF document you need to add some extra configuration.
+
+----
+dookug:
+  service:
+    engine:
+      pdf:
+        digitalsign:
+          signature:
+            sampleProfile:
+              name: Example Ltd.
+              reason: Certified
+              keystore: /home/icellmobilsoft/keys/keystore.p12 <3>
+              keystorePass: 123456
+              keystoreType: PKCS12
+              keyAlias: key_test
+              dss: <1>
+                imageFile: /home/icellmobilsoft/pdfsign/sample/nusz_signature.png <2>
+                page: 1 <3>
```

Comment:
kulcs nevekre nagyon sok mindenre lehet gondolni. Valahogy a konretizalni jo lenne, pl:
```
                showOnPage: 1 <3>
                position:
                  left: 18 <4>
                  top: 2 <5>
                width: 2 <6>
```
Egyebkent az nem latszik (vagy nincs leirva) hogy a pozicioja mihez kepest van gondolva. A centimeter is eleg szokatlan egyseg, szerintem tul nagy